docs/build_docs/source/tutorials/peleffy.rst

```diff
@@ -0,0 +1,167 @@
++---------------------------------+--------------+-------------------------------------------------------+
+| ``--chain CHAIN``               | string       | Chain ID of the molecule to parameterize.             |
++---------------------------------+--------------+-------------------------------------------------------+
+| ``--include_terminal_rotamers`` | None         | Do not exclude terminal rotamers when building        |
+|                                 |              |                                                       |
+|                                 |              | the rotamer library.                                  |
++---------------------------------+--------------+-------------------------------------------------------+
+
+**Example:** Parametrization of ligand contained in my_ligand.pdb file with openff-1.3.0 force field, rotamer resolution
+of 10 degrees, am1bcc charge calculation method and custom output directory.
+
+.. code-block:: bash
+
+    python -m peleffy.main my_ligand.pdb -f openff_unconstrained-1.3.0.offxml -o "ligand_params" -r 10 -c am1bcc
+
+API
+---
+
+If you are comfortable with python, you can make use of peleffy's API to parametrize your ligand. Follow the steps below
+to get a general idea how it works and for more details, please refer to `peleffy documentation <https://martimunicoy.github.io/peleffy/index.html>`_.
+
+1. The first step is always initializing a **Molecule object**, you can do it either from a PDB file or a string of SMILES
+
+.. code-block:: python
+
+    from peleffy.topology import Molecule
+
+    molecule = Molecule("my_ligand.pdb")  # from a PDB file
+    molecule = Molecule(smiles='c1ccc2cc3ccccc3cc2c1', hydrogens_are_explicit=False)  # from SMILES
+
+2. Generating **rotamer library**
+
+.. code-block:: python
+
+    # Create a rotamer library
+    rotamer_library = RotamerLibrary(molecule)
+    rotamer_library.to_file('LIG.rot.assign')
+
+3. **Parametrization** and creating the Impact file
+
+a. with OpenFF
+
+.. code-block:: python
+
+    from peleffy.forcefield import OpenForceField
+    from peleffy.topology import Topology
+    from peleffy.template import Impact
+
+    openff = OpenForceField('openff_unconstrained-1.2.0.offxml')
+    parameters = openff.parameterize(molecule)
+    topology = Topology(molecule, parameters)
+    impact = Impact(topology)
+    impact.to_file('ligz')
+
+b. with OPLS2005
+
+.. code-block:: python
+
+    from peleffy.forcefield import OpenForceField
+    from peleffy.topology import Topology
+    from peleffy.template import Impact
+
+    openff = OpenForceField('openff_unconstrained-1.2.0.offxml')
+    parameters = openff.parameterize(molecule, charge_method='OPLS2005')
+    topology = Topology(molecule, parameters)
+
+    impact = Impact(topology)
+    impact.to_file('ligz')
+
+4. Generating OBC **solvent parameters** (applicable only if using OpenFF)
```

Comment:
Actually, there is the possibility to generate OBC parameters that are compatible with OPLS with: https://martimunicoy.github.io/peleffy/api/autogenerated/peleffy.solvent.OPLSOBC.html#peleffy.solvent.OPLSOBC